terraform/lambda/ipv-session.tf

```diff
@@ -0,0 +1,17 @@
+module "ipv-session" {
+  source      = "../modules/endpoint"
+  environment = var.environment
+
+  rest_api_id            = aws_api_gateway_rest_api.ipv_internal.id
+  rest_api_execution_arn = aws_api_gateway_rest_api.ipv_internal.execution_arn
+  root_resource_id       = aws_api_gateway_rest_api.ipv_internal.root_resource_id
+  http_method            = "POST"
+  path_part              = "ipv-session"
+  handler                = "uk.gov.di.ipv.lambda.IpvSessionHandler::handleRequest"
+  function_name          = "${var.environment}-create-ipv-session"
```

Comment:
I think this PR also needs the amend we made to the bash script to deploy the new lambda function to localstack ?

Reply:
That will be a separate PR within the localstartup repo